quickstarts/microsoft.mobilenetwork/mobilenetwork-provision-proxy-sims/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "simResources": {
+            "value": [
+                {
+                    "simName": "sim-1",
+                    "integratedCircuitCardIdentifier": "8900000000000000001",
+                    "internationalMobileSubscriberIdentity": "000000000000001",
+                    "authenticationKey": "00112233445566778899AABBCCDDEEFF",
+                    "operatorKeyCode": "00000000000000000000000000000001",
+                    "deviceType": "None"
```

Comment:
Indentation in this file is off. Please fix.

Reply:
This file is generated via 'vs code'. Not addressed to have the same code style across the templates.